_posts/2016-04-08-foreman-community-newsletter-april-2016.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Foreman Community Newsletter - April 2016
+author: Amir Fefer
+tags:
+- foreman
+- newsletter
+excerpt: |
+  Foreman 1.11 and 1.10.3 RC2 released. Read on for details of new plugin releases and new recordings to watch!
+---
+
+### Foreman 1.11 release is out
+Foreman 1.11 was [announced](https://groups.google.com/forum/#!topic/foreman-announce/ZieXOPTC98Q) on 1st of April, and it's not April Fool. The headline features include Improvements to [host and host group](https://youtu.be/PXaVBB88b2Y) parameter editing, [Smart proxy status pages](https://youtu.be/BusFmWMsTbI) enhancements, Smart proxy [DHCP provider]() support, new [host bulk actions](https://youtu.be/xTQuF_KX95Q) and new test buttons for [LDAP and email](https://youtu.be/HP45Eo_JXC0) notifications.
+Check the full [release notes](http://theforeman.org/manuals/1.11/index.html#Releasenotesfor1.11)
+
+### Foreman 1.10.3 bugfix release is out
+On 31th of March Foreman 1.10.3 has been released with a low severity security fix and a number of bug fixes.
+For more details check the [release notes](http://theforeman.org/manuals/1.10/index.html#Releasenotesfor1.10.3)
+
+### Katello 3.0 RC1 release is out
+On 22th of March Katello 3.0 RC1 has been released. The highlights features include Foreman Remote Execution Integration, OSTree Content Support, Capsule Status Views,  Lazy Sync, Mirror on Sync,  Docker V2 Support and many more
+Check the full [release notes](http://www.katello.org/docs/3.0/release_notes/release_notes.html)
+
+Thanks to everyone who contributed to these releases - code, testing, bug reports, translations, your efforts are very much appreciated!
+
+### New and updated plugins since last March's newsletter
+ - [foreman_ansible_inventory](https://github.com/theforeman/foreman_ansible_inventory) - updated to 0.0.3
+ - [foreman-discovery](https://github.com/theforeman/foreman_discovery) updated to 5.0.2
+ - [hammer-cli-foreman](https://github.com/theforeman/hammer-cli-foreman/tree/0.6.2) updated to 0.6.2
+ - New plugin [smart_proxy_dns_infoblox](https://github.com/sjoeboo/smart_proxy_dns_infoblox) released
```

Comment:
Yeah, foreman-cockpit 2.0.1 was released yesterday. I tried to summarize the changes here - https://groups.google.com/forum/#!topic/foreman-users/3hVeZBDkdyU - but summing up: Cockpit component URLs were not 'stabilized' until now. Thus, modern versions of Cockpit (0.78+) use a different set of URLs that the plugin now supports.